VMT/VMT.md

```diff
@@ -0,0 +1,225 @@
+# Vulnerability Management Process
+
+The Kata Containers Vulnerability Management Team (VMT) is responsible for coordinating the progressive disclosure of a vulnerability.
+
+Members of the team are independent and security-minded folks who ensure that vulnerabilities are dealt with in a timely manner and that downstream stakeholders are notified in a coordinated and fair manner.
+Where a member of the team is employed by a downstream stakeholder, the member does not give their employer prior notice of any vulnerabilities.
+In order to reduce the risk of accidental disclosure of vulnerability in the early stages, membership of this team is intentionally limited to a small number of people.
+
+# Acronyms
+
+To aid reading this document it helps to understand some key acronyms:
+
+| Acronym  | Description                                                     |
+| -------- | --------------------------------------------------------------- |
+| CNA      | [CVE Numbering Authority](https://cve.mitre.org/cve/cna.html)   | 
+| CVE      | [Common Vulnerabilities and Exposures](https://en.wikipedia.org/wiki/Common_Vulnerabilities_and_Exposures) |
+| KCSA     | Kata Containers Security Announcement                           |
+| KCSN     | Kata Containers Security Note                                   |
+| KCST     | Kata Containers Security Team                                   | 
+| PR       | [A GitHub Pull Request](https://help.github.com/articles/about-pull-requests/) |
+| VMT      | The Kata Containers Vulnerability Management Team               |
+
+# Supported versions
+
+> **FIXME** Before we can make statements about which stable versions we fix, we need to define the Kata Containers release and maintenance process.
+
+The VMT coordinates patches fixing vulnerabilities in supported stable branches of Kata Containers, in addition to the master branch (next version under development).
+
+# Process
+
+Each security bug is assigned a VMT *coordinator* (a member from the VMT) that will drive the fixing and disclosure process.
+Here are the steps we follow.
+
+![VMT process workflow](VMT-process.png)
+
+## Reception
+
+A report is received via an email to the [VMT mailing list](mailto:kata-vmt@lists.katacontainers.io).
+
+> **FIXME** We need to decide where and how reports are handled.  The obvious choice is to use a private GitHub repo alongside the other Kata Containers repos - but, we would have to pay for that (private repos are not free).  Another alternative may be to use any OSF infrastructure already in place and in use for other OSF projects, if any exists.
```

Comment:
Assuming that we stay on GitHub, I believe our best avenue for reporting is directly to the VMT themselves. If we switch to a different bug tracker we might have more private disclosure options, but I think we'll need to list name, email and key similar to [how the K8S vulnerability team does.](https://github.com/kubernetes/sig-release/blob/master/security-release-process-documentation/security-release-process.md#product-security-team-pst)

Reply:
Right, reporting goes direct to the VMT. That can either be to the mailing list (yet to be created), or it could be direct to a member of the VMT, in which case, yes we need to publish who is on the list, and probably their keys.

The bit we don't currently have a solution for is then how does the VMT, and the reporter, track progress of the report. We don't want to (cannot) store them in public github Issues.
Seeing that OS uses Launchpad afaict, does that mean there maybe some infra inside the OSF we can duplicate/leverage for re-use with kata VMT Issues? @annabellebertooch

Reply:
Most OSF projects use either Launchpad or StoryBoard, both of which have support for a private bug workflow. The way we normally handle reports of suspected vulnerabilities is that the reporter creates a private bug detailing the issue with the VMT is subscribed, who confirm it's reported against the correct project and then in turn subscribe relevant subject matter experts for that project (including people who are trusted to pre-approve patches to the source code pending their eventual publication). Discussion and proposed solutions/patches are posted into the private bug and, at some point depending on how the discussion unfolds, the bug is switched to become public along with patches pushed into public code review and, if relevant, a security advisory issued at the same time referring both to the now public bug report and fixes under final review.

Reply:
Much though I'd like to not have another place to log Issues for Kata, but:
- Given we very likely won't be using private GitHub repos
- And other OS[F] projects are already on launchpad
- and, Launchpad is free for open source projects, and has a private issue ability...

I'm going to make the suggestion that we set up a launchpad.net kata project, and use that **only** for VMT vulnerability reporting. thoughts?

Reply:
@devimc has already created one here:

- https://launchpad.net/kata-containers